_posts/0000-01-02-leon770110.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
**Standing on the shoulders of giants** https://ofabondservant.com/2018/06/15/standing-on-the-shoulders-of-giants/

Reply:
This is good